docs/stories/components/charts/LLinearChart.stories.js

```diff
@@ -11,21 +11,23 @@ export default {
     isExpanded: { control: 'boolean', description: 'Chart expanded: true / Chart minimized: false' },
     isExpandable: { control: 'boolean', description: 'Chart expandable: true / Chart not expandable: false' },
     expandList: { action: 'expandList', description: 'Toggle chart expanded when "See more" clicked' },
+    eventClick: { action: 'eventClick', description: 'Event emitted activated when label is clicked' },
```

Comment:
Sugestão de alteração para description: 'Event emitted when label is clicked" (sem o activated)